ios/Codeforces Watcher/Application/features/users/cells/sectionTitle/TitleSectionTableViewCell.swift

```diff
@@ -0,0 +1,31 @@
+import SwiftUI
+
+class TitleSectionTableViewCell: UITableViewCell {
```

Comment:
And once we migrate the full screen to `SwiftUI` will be we able to get rid of this wrapper?

Reply:
Yes, UITableViewCell reuse has been replaced with SwiftUI List, but I have yet to explore the current UIKit implementation to rewrite it correctly, so step by step.